### ISSUE TITLE
Issue with hover display on image vs result array

### ISSUE THREAD

**Issue by nrobert** [PeterOrneholm/cognitive-workbench#13]
The image metadata which is displayed on hover (for example on face) is hidden by the result array on the right.

It should be displayed on top of this array.

Sample:
![image](https://user-images.githubusercontent.com/6305666/77901558-11e7b680-7280-11ea-881f-f98f8fe35464.png)

**Comment by PeterOrneholm** [comment]
Good point. Maybe it's just some z-index, or we have to rearrange the columns a bit.
I can try to have a look at it.

**Comment by nrobert** [comment]
If you can have a look, it might be faster than me, I really hate css issues!
